automatic/firefox/tools/chocolateyInstall.ps1

```diff
@@ -17,26 +19,65 @@ if (Get-32bitOnlyInstalled -product $softwareName) {
   )
 }
 
+$args = ""
+
+# Command Line Options from the Firefox installer
+# https://firefox-source-docs.mozilla.org/browser/installer/windows/installer/FullConfig.html
+
+# Always prevent Firefox installer to require a reboot
+$args = $args + " /PreventRebootRequired=true"
+
+# Prevent RemoveDistributionDir by default
+$args = $args + " /RemoveDistributionDir=false"
+
+
+if ($PackageParameters['InstallDir']) {
+  $args = $args + " /InstallDirectoryPath=" + $PackageParameters['InstallDir']
+}
+
+if ($PackageParameters.NoTaskbarShortcut) {
+  $args = $args + " /TaskbarShortcut=false"
+}
+
+if ($PackageParameters.NoDesktopShortcut) {
+  $args = $args + " /DesktopShortcut=false"
+}
+
+if ($PackageParameters.NoStartMenuShortcut) {
+  $args = $args + " /StartMenuShortcut=false"
+}
+
+if ($PackageParameters.NoMaintenanceService) {
+  $args = $args + " /MaintenanceService=false"
+}
+
+if ($PackageParameters.RemoveDistributionDir) {
+  $args = $args + " /RemoveDistributionDir=true"
+}
+
+if ($PackageParameters.NoAutoUpdate) {
+  $args = $args + " /MaintenanceService=false"
+}
+
 if ($alreadyInstalled -and ($env:ChocolateyForce -ne $true)) {
```

Comment:
Not correct  because env vars do not support powershell boolean vars.  Just use if `(... -and $Env:ChocolateyForce)`

Reply:
thanks, this is fixed now